deploy/examples/dashboards/SimpleDashboard.yaml

```diff
@@ -6,6 +6,7 @@ metadata:
     app: grafana
 spec:
   name: simple-dashboard.json
+  environment: env-4
```

Comment:
How will we deploy it on other env? Does it mean that we will need to modify dashboard for every env?

Reply:
This is the sample dashboard CR k8s manifest, we currently deploy dashboards via helm charts. In helm charts, we can use variables that can be substituted by a particular value from `values.yaml`, something like `environment: {{ .Values.env }}`.
But, we have to modify the dashboard helm chart.